libtransmission/transmission.h

```diff
@@ -512,20 +513,20 @@ bool tr_sessionIsSpeedLimited(tr_session const*, tr_direction);
 ****  Alternative speed limits that are used during scheduled times
 ***/
 
-void tr_sessionSetAltSpeed_KBps(tr_session*, tr_direction, unsigned int kilo_per_second);
-unsigned int tr_sessionGetAltSpeed_KBps(tr_session const*, tr_direction);
+void tr_sessionSetAltSpeed_KBps(tr_session*, tr_direction, tr_speed_t kilo_per_second);
+tr_speed_t tr_sessionGetAltSpeed_KBps(tr_session const*, tr_direction);
 
 void tr_sessionUseAltSpeed(tr_session*, bool);
 bool tr_sessionUsesAltSpeed(tr_session const*);
 
 void tr_sessionUseAltSpeedTime(tr_session*, bool);
 bool tr_sessionUsesAltSpeedTime(tr_session const*);
 
-void tr_sessionSetAltSpeedBegin(tr_session*, int minutes_since_midnight);
-int tr_sessionGetAltSpeedBegin(tr_session const*);
+void tr_sessionSetAltSpeedBegin(tr_session*, time_t minutes_since_midnight);
+time_t tr_sessionGetAltSpeedBegin(tr_session const*);
```

Comment:
`time_t` represents seconds, so it's confusing / misleading / error-prone to have it count minutes. This should be an `unsigned int`

Reply:
I see. In general, seconds, minutes and hours are all stored as `long`, but since that specific usage here is capped to maximum 24h, you would prefer it stored in a smaller type. A little bit like the difference between `chrono::microseconds` which is a `long long` and `suseconds_t` which is an `int32_t` because it's capped to the maximum number of microseconds in one second.

I'll do this change, although it might mean many static_casts.